iscsidev/iscsi.go

```diff
@@ -180,6 +181,29 @@ func (dev *Device) StopInitiator() error {
 	return nil
 }
 
+func (dev *Device) RefreshInitiator() error {
+	lock := nsfilelock.NewLockWithTimeout(util.GetHostNamespacePath(HostProc), LockFile, LockTimeout)
+	if err := lock.Lock(); err != nil {
+		return fmt.Errorf("Fail to lock: %v", err)
```

Comment:
`return errors.Wrap(err, "failed to lock")``

Could you also fix other errors here like `errors.Wrapf(err, "failed to lock")` -> `errors.Wrap(err, "failed to lock")`